pkg/services/dashboards/service/dashboard_service.go

```diff
@@ -319,7 +319,7 @@ func (dr *DashboardServiceImpl) SaveProvisionedDashboard(ctx context.Context, dt
 
 	dto.User = accesscontrol.BackgroundUser("dashboard_provisioning", dto.OrgID, org.RoleAdmin, provisionerPermissions)
 
-	cmd, err := dr.BuildSaveDashboardCommand(ctx, dto, dr.cfg.IsLegacyAlertingEnabled(), false)
+	cmd, err := dr.BuildSaveDashboardCommand(ctx, dto, false, false)
```

Comment:
as long as we delete the flag, probably it makes sense to either leave a \\TODO or refactor the method and remove the flag at all?

Reply:
We will remove it in a follow up